maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -97,6 +97,8 @@
 import javax.annotation.Nonnull;
 import java.io.File;
 import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
Sorry, I did not tell you that we use reflection utility class `ReflectionUtils` which simplifies all the troubles around the reflection. Pls use that. Thx

Reply:
I just took the code from compiler plugin.  Are you saying that it's wrong there too?

Reply:
compiler does not call surefire's utilities. We use it a lot and therefore these utilities cope with annoying exceptions, etc.

Reply:
Changed to ReflectionUtils, although it doesn't seem to have made the code much shorter... unless I did it wrong